### DISCUSSION TITLE
The YouTube Videos

### DISCUSSION THREAD

**Discussion by Five-Birds** [rsxdalv/TTS-WebUI#70] [General]
Please remove the two videos you provided,as the content creator of the videos is providing a link in which he says necessary information is contained how to install and use the WebUI. But the link is directing to a website (shrinke.me/TTSdescription) which is full of malware and adware. Even the content he promised is not there. Comments which warn other user about this are deleted. Also i tried to link this GitHub rep as a comment, buit it is also deleted by the content creator.

So in my opinion, the guy is feeding people with the webui as a honeypot to trick people to the malware site. This reflects no good light to your excellent project!

**Comment by rsxdalv** [comment]
Got it, I'll remove it, thanks!

**Reply by Five-Birds** [reply]
Thank you. It is really bad. He is explaining the process of installtion and use very good, but than does this shit with the shrinke.me link...

**Comment by rsxdalv** [comment]
Done
https://github.com/rsxdalv/tts-generation-webui/commit/1c24f4fc490c8c26db753daf85b8ef0bdae80878
I can see, it's probably a click farm that's taken several steps too far.
I'm glad that you reported this.
Just for reference, I wouldn't mind if that was something within reason, to support the video creator. But this case is a bit out of scope.
(I tried closing the "popups", but even after 10-20 clicks it didn't lead me anywhere, only gambling and suspicious sites. In the end, I am not sure if it's even possible to get to the actual content.).

**Comment by Five-Birds** [comment]
Hi Robert,

Thank you. I also tried to get to the content of the guy and managed it to go behind the clicks… but than a “no document is available”. I also reported the videos to YouTube.

 

Btw.: Excellent work on your WebUI – All in one hand and working really good. Thank you for your work!

 

BR Dirk

 

Von: Roberts Slisans ***@***.***> 
Gesendet: Mittwoch, 12. Juli 2023 22:25
An: rsxdalv/tts-generation-webui ***@***.***>
Cc: Five-Birds ***@***.***>; Author ***@***.***>
Betreff: Re: [rsxdalv/tts-generation-webui] The YouTube Videos (Discussion #70)

 

Done
 <https://github.com/rsxdalv/tts-generation-webui/commit/1c24f4fc490c8c26db753daf85b8ef0bdae80878> 1c24f4f
I can see, it's probably a click farm that's taken several steps too far.
I'm glad that you reported this.
Just for reference, I wouldn't mind if that was something within reason, to support the video creator. But this case is a bit out of scope.
(I tried closing the "popups", but even after 10-20 clicks it didn't lead me anywhere, only gambling and suspicious sites. In the end, I am not sure if it's even possible to get to the actual content.).

**Reply by rsxdalv** [reply]
So interesting thing - he fixed the links at some point.
